Tests/Shared/UnitTest.swift

```diff
@@ -66,7 +66,7 @@ open class UnitTest: XCTestCase {
         setUp(enableLogs: false)
     }
 
-    public func setUp(enableLogs: Bool = false, modifySdkConfig: ((inout SdkConfig) -> Void)? = nil) {
+    open func setUp(enableLogs: Bool = false, modifySdkConfig: ((inout SdkConfig) -> Void)? = nil) {
```

Comment:
more for my curiosity whats the diff between `open & public` and why change it

Reply:
In Swift, functions/classes are `final` by default meaning they can't be overriden. 

`UnitTest` functions are overridden by other abstract classes so I changed it to `open` so `setUp` can be overriden by `IntegrationTest` class.